plugins/repository-gcs/src/test/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStoreRepositoryTests.java

```diff
@@ -199,6 +199,8 @@ StorageOptions createStorageOptions(final GoogleCloudStorageClientSettings clien
                                                     final HttpTransportOptions httpTransportOptions) {
                     StorageOptions options = super.createStorageOptions(clientSettings, httpTransportOptions);
                     return options.toBuilder()
+                        .setHost(options.getHost())
```

Comment:
For some reason the `toBuilder` keeps track of all settings but these two in the latest version so we have to manually set these again.